src/app/services/user-service.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
```

Comment:
I'm using hardcoded data, I'll update the service to use the firebase data in the next commit, you can review the design on the register user formulary.